flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/dialect/clickhouse/ClickhouseDialectFactory.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.flink.connector.jdbc.dialect.clickhouse;
+
+import org.apache.flink.annotation.Internal;
+import org.apache.flink.connector.jdbc.dialect.JdbcDialect;
+import org.apache.flink.connector.jdbc.dialect.JdbcDialectFactory;
+
+/** */
```

Comment:
well it's better to have some description about the class rather than an empty comment